server/metrics/metrics.go

```diff
@@ -2224,6 +2224,16 @@ var (
 		GroupID,
 	})
 
+	PebbleCacheNumCDCChunks = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: bbNamespace,
+		Subsystem: "remote_cache",
+		Name:      "pebble_cache_num_cdc_chunks",
```

Comment:
Should we call this something like chunks_per_file_count? The name sounds like it's counting total number of chunks in the cache.

Reply:
Good point. I named it like pebble_cache_num_chunks_per_file instead. Trying to avoid "_count" at the end b/c it might sound like a Count metrics instead of a Histogram.